### ISSUE TITLE
Update and Add Programs

### ISSUE THREAD

**Issue by c-sanchez** [npackd/npackd-cpp#69]
Hi, Npackd is really great, it is the only one of its kind, I thought I would not find anything like it seeing that all alternatives to "winget" are basically the same, at least in my opinion.
This is the only one that is really portable and offers a nice, simple and fast interface and doesn't force you to use the command line at all if you don't want to.

On the other hand, like any package manager, it won't have everything we might need all the time, although Npackd surprised me with everything it has.

Could you update and add the following programs in the repository?

**Update**
> AkelPad to 4.9.12 (from ssrlive)
> https://github.com/ssrlive/akelpad/releases
> 
> Audacity to 3.5.1
> https://www.audacityteam.org/download/windows/
> 
> Cent Browser to 5.1.1130.129
> http://www.centbrowser.com/history.html
> 
> Free Download Manager Ultra-Light (FDM-UL) - 1.1
> https://github.com/59de44955ebd/FDM-UL/releases
> 
> Free Shooter to 2.1.1
> https://github.com/henrypp/freeshooter/releases
> 
> Inkscape to 1.3.2
> https://inkscape.org/release/inkscape-1.3.2/
> 
> KeePassXC to 2.7.9
> https://github.com/keepassxreboot/keepassxc/releases
> 
> MPC-HC to 2.3.2
> https://github.com/clsid2/mpc-hc/releases
> 
> SumatraPDF to 3.5.2
> https://www.sumatrapdfreader.org/download-free-pdf-viewer
> 

**Add**
> 3D Youtube Downloader - 1.20.4
> https://yd.3dyd.com/download/
> 
> 3D Youtube Downloader Batch - 2.13.3
> https://ba.3dyd.com/download/
> 
> Alternate File Move - 2.520
> https://www.alternate-tools.com/pages/c_filemove.php?lang=ENG
> 
> ares - 139
> https://github.com/ares-emulator/ares/releases
> 
> Chromium (Stable version Builds) - 126.0.6478.62
> https://github.com/Hibbiki/chromium-win64/releases
> 
> Delta Patcher 3.1.5
> https://github.com/marco-calautti/DeltaPatcher/releases
> 
> Discord Messenger - 1.06
> https://github.com/DiscordMessenger/dm/releases
> 
> FileCentipede - 2.83
> https://github.com/filecxx/FileCentipede/releases
> 
> FreeFileSync 13.7
> https://freefilesync.org/download.php
> 
> Godot - 4.2.2
> https://github.com/godotengine/godot/releases
> 
> HiBit Uninstaller - 3.2.20
> https://www.hibitsoft.ir/Uninstaller.html
> 
> HTTP Downloader - 1.0.6.2
> https://github.com/erickutcher/httpdownloader/releases
> 
> LocalSend - 1.14.0
> https://github.com/localsend/localsend/releases
> 
> Miranda NG - 0.96.5
> https://miranda-ng.org/downloads/ 
> 
> Nestopia UE - 1.52.1
> https://github.com/0ldsk00l/nestopia/releases
> 
> NitroShare - 0.3.4
> https://github.com/nitroshare/nitroshare-desktop/releases
> 
> RapidCRC Unicode - 0.3.40
> https://www.ov2.eu/programs/rapidcrc-unicode
> 
> StemRoller 2.0.6
> https://github.com/stemrollerapp/stemroller/releases
> 
> Wings 3D - 2.3
> https://sourceforge.net/projects/wings/files/wings/
> 
> Wise Disk Cleaner - 11.1.3
> https://www.wisecleaner.com/wise-disk-cleaner.html
> 
> Wise Registry Cleaner - 11.1.5
> https://www.wisecleaner.com/wise-registry-cleaner.html
> 
> Xtreme Download Manager - 8.0.29
> https://github.com/subhra74/xdm/releases

If there was something to criticize about the program, personally it would be the xml format used for the repository.
I think it is a horrible format to read, but I understand why it is used.
Just as a mere curiosity, wasn't it possible to use something simpler to read and use?

**Comment by tim-lebedkov** [comment]
Please add new packages here yourself: https://npackd.appspot.com/package/new

**Comment by tim-lebedkov** [comment]
AkelPad updated

**Comment by c-sanchez** [comment]
> Please add new packages here yourself: https://npackd.appspot.com/package/new

Yeah, I was confused with the other website, https://www.npackd.org/

I've been looking a bit at how to add packages, great that we have the option to collaborate that way :D
I did it first with 3D Youtube Download to test.
I will collaborate as I can adding more packages :)

> AkelPad updated

Thanks! :D

**Comment by tim-lebedkov** [comment]
Another (future) application was running on npackd.org. Now the two domains are the same again.

**Comment by tim-lebedkov** [comment]
Re. XML: I like it. It is very readable to me. Readability is not enough to justify a change of the repository format.
